nc_time_axis/tests/integration/test_plot.py

```diff
@@ -1,43 +1,46 @@
 """Integration test for plotting data with non-gregorian calendar."""
 
-from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, zip)  # noqa
-
 import unittest
 
 import matplotlib
-matplotlib.use('agg')
-import matplotlib.pyplot as plt  # nopep8
-import cftime  # nopep8
-import numpy as np  # nopep8
 
```

Comment:
Why the empty line here (line 6 of the new file, not line 13 of the old)? Is this from isort?

Reply:
Yeah, this is `isort` injected.

We can customise `isort` to make it do what we want, but I figure what we get out the box is good enough.

Reply:
It just seems a bit inconsistent with the ordering. I noticed in a further change (https://github.com/SciTools/nc-time-axis/blob/08caec3f7e6fc9a26a5147d0d58f018ca93a4441/nc_time_axis/tests/unit/test_NetCDFTimeDateLocator.py#L8-L13)

The matplotlib.style.use() call is separated from the imports, so I wondered if it was trying to do that here, and then accidentally creating the extra, empty line. But I guess it's not really a big deal, just a bit confusing for me as to what it is trying to do

Reply:
`matplotlib.style.use("classic")` is a statement in the global namespace and not an import.

I think that's okay here... or do you think there is an issue?

Reply:
>matplotlib.style.use("classic") is a statement in the global namespace and not an import.
Yep same as matplotlib.use. We want to call matplotlib.use prior to import pyplot. I was just trying to understand why the extra, empty line.